README.md

```diff
@@ -1,8 +1,228 @@
-# orchard
+# Orchard
 
 > An intentional plantation of trees or shrubs that is maintained for food production.
 
 [![CircleCI](https://circleci.com/gh/salesforce/orchard.svg?style=svg)](https://circleci.com/gh/salesforce/orchard)
 
-Orchard is an orchestration service that manages workflows and activities
-inspired by AWS Data Pipeline
+Orchard is an orchestration service that manages data pipelines, compute workflows, associated ETL/ELT activities, AND manages the underlying resource lifecycle (provisioning, monitoring, termination).
+Inspired by AWS' [Data Pipeline](https://aws.amazon.com/datapipeline/) service, Orchard is designed for enterprise use-cases that demand security, extreme concurrency, granular control over the resource lifecycle, and flexible integration with a cloud-based microservice architecture.
+
+## Design
+Like Apache Spark, Orchard is written in functional Scala. This gives Orchard the power of Scala's well-developed concurrency features, and in particular, the Actor pattern as enabled by Scala's [Akka](https://github.com/akka/akka) library.
+
+## Setup
+Orchard is designed to be deployed into a cloud environment as a service, but can alternatively be set up locally for exploration and development. To do so, follow these steps. 
+
+**Install Apps (OSX)**
+```sh
+# clone orchard into a local directory
+git clone git@github.com:salesforce/orchard.git
+
+# use brew to install Scala Build Tool (SBT)
+brew install sbt
```

Comment:
I think we would like to recommend sdkman for sbt instead of brew's version because of https://github.com/Homebrew/homebrew-core/issues/50649

Reply:
Ah that makes sense.